lib/spawn-panel.js

```diff
@@ -0,0 +1,15 @@
+const spawn = require("cross-spawn");
+const mv = require("multiview")();
+const path = require("path");
+
+module.exports = async function run(projDir) {
```

Comment:
no need of async, you are not using await anywhere and you are returning a promise